straxen/corrections_services.py

```diff
@@ -74,7 +74,11 @@ def get_corrections_config(self, run_id, correction=None, config_model=None):
         model_type, global_version = config_model
 
         if correction == 'pmt_gains':
-            return self.get_pmt_gains(run_id, model_type, global_version)
+            return self.get_pmt_gains(run_id, model_type, global_version, 'tpc')
```

Comment:
Perhaps I'm overseeing something, but would the `global_version` not already make sure that you distinguish between tpc, MV and NV? I think you can keep the code somewhat simpler in this case by not having to add a new argument.

Reply:
As in I assume it's now:
```python
#TPC 
   gain_model=("CMT_model", ("to_pe_model", "ONLINE")),
# MV
   gain_model=("CMT_model", ("to_pe_model_NV", "ONLINE")),
# NV
   gain_model=("CMT_model", ("to_pe_model_MV", "ONLINE")),
```
Otherwise I think it should be like this. If the global version is already like this, there is no need to write 3 extra `elif`-statements right?

Reply:
see comment below

Reply:
Aaron, I think I do not understand your answer to Joran comment. Why is it not possible to change the code as Joran propose? Indeed, I also think it is more elegant to have a structure such as:
```
Version, Online:
          | -> to_pe_tpc
          |-> to_pe_nv
          |-> elife
Version SR0:
          |-> to_pe_tpc .... 
```

Reply:
I think I(we) was(were) confused by the comment, see below